src/main/java/nl/nn/testtool/Config.java

```diff
@@ -101,6 +116,31 @@
 @Lazy // Lazy init singleton beans (prototype beans are already loaded on demand)
 @Configuration
 public class Config {
+	@Bean
+	OpenTelemetry openTelemetry() {
+		Resource resource = Resource.getDefault().toBuilder().put(ServiceAttributes.SERVICE_NAME, "ladybug").put(ServiceAttributes.SERVICE_VERSION, "1.0.0").build();
+		String endpointZipkin = "http://localhost:9411/api/v2/spans";
```

Comment:
Shouldn't this be configurable? Does this code always enable Zipkin and Jaeger? Shouldn't it be configurable whether or not they should be enabled?

Reply:
I think I can't make this bean configurable, but I can make a condition in the other code to check if telemetry data have to be sent. If it's no, this bean will also not be used.

Reply:
You can make the a separate bean of String endpointZipkin and check whether this bean is present or not to determine whether or not to use Zipkin and do something similar for Jaeger and have openTelemetry() return null when both are not configured. Wherever you use testTool.getOpenTelemetryTracer() you need to check whether it is null and do nothing when this is the case.

And then for example in the test webapp you can configure one of them in https://github.com/wearefrank/ladybug-test-webapp/blob/master/src/main/resources/springTestToolTestWebapp.xml by adding for example:

		<bean name="endpointZipkin" class="java.lang.String">
			<constructor-arg value="http://localhost:9411/api/v2/spans"/>
		</bean>

And something similar for Jaeger. Please add them in comment so pleople can enable them when needed and document it in the README.